api/ordinals/src/pg/brc20/brc20-pg-store.ts

```diff
@@ -121,6 +122,36 @@ export class Brc20PgStore extends BasePgStore {
     };
   }
 
+  async getTransferableInscriptions(
+    args: {
+      address: string;
+      ticker?: string[];
+    } & DbInscriptionIndexPaging
+  ): Promise<DbPaginatedResult<DbBrc20TransferableInscription>> {
+    const results = await this.sql<(DbBrc20TransferableInscription & { total: number })[]>`
+      SELECT
+        o1.inscription_number, o1.inscription_id, o1.block_height, o1.amount, o1.ticker, COUNT(*) OVER() AS total
+      FROM operations AS o1
+      WHERE o1.operation = 'transfer'
+      AND o1.address = ${args.address}
+      ${args.ticker ? this.sql`AND o1.ticker IN ${this.sql(args.ticker)}` : this.sql``}
```

Comment:
You should compare the lowercased version of the given tickers... I think we have an example of that somewhere else in this file